#!/usr/bin/python3
def raise_exception():
    a = 16
    b = "seize, lol"
    return a + b
